src/jsmodules/add.js

```diff
@@ -0,0 +1,20 @@
+import { uPStoreValues } from './storevalues.js';
+import { ul } from './selectors.js';
+
+export class TaskAdd {
+  constructor(task, bool, id) {
+    this.task = task;
+    this.bool = bool;
+    this.id = id;
```

Comment:
- [ ] Let's talk about these two task properties.
* Firstly, a more semantic way to call this property defined as `bool` would be `completed` just like in the requirements. `bool` is too generic and does not give enough meaning as per the context of a task. Think about `task.completed` VS `task.bool`, which one sounds better? consider renaming it to **`completed`**.
* Secondly, a task item should also have the property of `index` which I assume you've defined as `id`. I suggest you rename this one too.

Let us take a look at the requirement below:

> **- By default new tasks should have the property `completed` set to `false` and the property `index` set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).**

Reply:
I renamed all the variables.